primefaces/src/main/resources/META-INF/resources/primefaces/forms/forms.selectcheckboxmenu.js

```diff
@@ -1044,6 +1063,8 @@ PrimeFaces.widget.SelectCheckboxMenu = PrimeFaces.widget.BaseWidget.extend({
                     $this.postHide();
                 }
             });
+
+            this.keyboardTarget.focus();
```

Comment:
Don't call `.focus()` that is deprecated jQuery call `.trigger('focus')` that is the recommended way now

Reply:
has been added